assets/performanceprofile/configs/99-workload-partitioning.conf

```diff
@@ -0,0 +1,12 @@
+{{- /* 
```

Comment:
I think it is time we add the intended destination for these configs if we can. This is a crio.conf.d file, right?

Reply:
Yup, this is a crio file, @MarSik do you mean that we should have this file in `assets/performanceprofile/configs/crio/` in the code, or are you suggesting we just add to the comment that we should mention this file will be created in `crio.conf.d`?

Reply:
Added it as a comment to the file, just incase.

Reply:
Just a comment is enough for me.